spec/rubocop/cop/style/stderr_puts_spec.rb

```diff
@@ -8,11 +8,11 @@
   it "registers an offense when using `$stderr.puts('hello')`" do
     expect_offense(<<-RUBY.strip_indent)
       $stderr.puts('hello')
-      ^^^^^^^^^^^^ Use `warn` instead of `$stderr.puts` to allow such output to be disabled.
+      ^^^^^^^^^^^^ Use `warn` instead of `$stderr.puts` or `STDERR.puts` to allow such output to be disabled.
```

Comment:
A very minor thing, but adding the word "either" here might help ("instead of either").

Reply:
Thanks. Text updated.